.gitattributes

```diff
@@ -18,6 +18,8 @@ CONTRIBUTING.md          ocaml-typo=missing-header
 INSTALL                  ocaml-typo=missing-header
 LICENSE                  ocaml-typo=non-printing,missing-header
 
+Makefile.shared          ocaml-typo=long-line
```

Comment:
Don't do that. We have very good reasons to limit lines to 80 characters and exceptions should be rare.